docs/translating/german.md

```diff
@@ -129,11 +129,7 @@ speakers in the community. It's all about making Zulip friendly and usable.
 
 _"Nachricht" (Facebook, WhatsApp, Transifex)_
 
-- Private Message (PM) - **Private Nachricht (PN)**
-
-Since we try to avoid concatenating words whenever possible, don't use
-"Privatnachricht" . PN is the officially used abbreviation for
-"Private Nachricht" and is used in many German chat forums.
```

Comment:
I think we should double-check whether we want to remove this part or update it since the current translation might not have followed this guideline. Can you please ask about this in the [German translation thread](https://chat.zulip.org/#narrow/stream/58-translation/topic/German.20translation) on CZO?